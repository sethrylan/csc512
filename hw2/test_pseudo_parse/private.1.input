VAR a,b:REAL;
BEGIN
a:=-1.0+0.20;
IF b>1 THEN WRITE(PRIVATE); ENDIF;
a:=110;
END
