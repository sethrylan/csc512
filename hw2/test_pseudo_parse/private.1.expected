syntax error in line 4 at PRIVATE
